### PR TITLE
Update Soundux to 0.2.7

### PR DESCRIPTION
**Pull request by D3SOX**

### CODE REVIEW

**Review by wizard-28 — COMMENTED** [review]
`pipewire-git` i.e. `pipewire` isn't a hard dependency of soundux, so why is it included here in the `pacdeps` array?

Citation:
image

File: packages/soundux/soundux.pacscript [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-version="0.2.6"
-url="https://github.com/Soundux/Soundux/releases/download/0.2.6/soundux-0.2.6.tar.gz"
+version="0.2.7"
+url="https://github.com/Soundux/Soundux/releases/download/0.2.7/soundux-0.2.7.tar.gz"
 build_depends="build-essential cmake libx11-dev libxi-dev libwebkit2gtk-4.0-dev libappindicator3-dev libssl-dev libpulse-dev"
 depends="libgtk-3-0 libwnck-3-0 libappindicator3-1 libwebkit2gtk-4.0-37 libx11-6 pulseaudio pulseaudio-utils"
 description="A cross-platform soundboard"
-hash="0cffff520b3d1e69ccbb10599ff1ff3c911d74eb01a4af8dc3d5132fa5535c63"
+hash="017003fc96f49df30575975f3904c0d8a500e325a9d2bca8c3dc69fed0cab0a7"
 maintainer="Soundux developers <info@soundux.rocks>"
```
Comment (wizard-28) [inline packages/soundux/soundux.pacscript:9]: Shouldn't the maintainer field have your name?
I mean you are the person who is maintaining this pacscript, not the whole organization.

**Comment by wizard-28** [comment]
@Henryws I think there is a problem with the `pacdeps` array. I think pacstall is not downloading the actual package (here: `soundux-0.2.7.tar.gz`) after pacstall installes the `pacdeps` dependencies.

**Comment by D3SOX** [comment]
> `pipewire-git` i.e. `pipewire` isn't a hard dependency of soundux, so why is it included here in the `pacdeps` array?
> 
> Citation:
> image

You must either have PulseAudio or PipeWire >= 0.3.26 for the program to function properly. However the PipeWire development headers are needed to compile it.

File: packages/soundux/soundux.pacscript [hunk 2]
```diff
@@ -1,18 +1,14 @@
 name="soundux"
 pkgname="soundux"
```
Comment (wizard-28) [inline packages/soundux/soundux.pacscript:2]: Actually this field is unnecessary too... I'll update `make-a-pacscript.md`

**Review by wizard-28 — COMMENTED** [review]
Well, we have to wait for `pacdeps` array to get fixed, and also for #13  to get merged, otherwise this is good

**Comment by Henryws** [comment]
> @Henryws I think there is a problem with the `pacdeps` array. I think pacstall is not downloading the actual package (here: `soundux-0.2.7.tar.gz`) after pacstall installes the `pacdeps` dependencies.

I'll check

**Comment by Henryws** [comment]
> Well, we have to wait for `pacdeps` array to get fixed, and also for #13 to get merged, otherwise this is good

It's working for me

File: packages/soundux/soundux.pacscript [hunk 3]
```diff
@@ -1,18 +1,13 @@
...
+version="0.2.7"
+url="https://github.com/Soundux/Soundux/releases/download/0.2.7/soundux-0.2.7.tar.gz"
 build_depends="build-essential cmake libx11-dev libxi-dev libwebkit2gtk-4.0-dev libappindicator3-dev libssl-dev libpulse-dev"
 depends="libgtk-3-0 libwnck-3-0 libappindicator3-1 libwebkit2gtk-4.0-37 libx11-6 pulseaudio pulseaudio-utils"
 description="A cross-platform soundboard"
-hash="0cffff520b3d1e69ccbb10599ff1ff3c911d74eb01a4af8dc3d5132fa5535c63"
-maintainer="Soundux developers <info@soundux.rocks>"
+hash="017003fc96f49df30575975f3904c0d8a500e325a9d2bca8c3dc69fed0cab0a7"
+maintainer="D3SOX <d3sox@protonmail.com>"
 pacdeps=("pipewire-git")
```
Comment (wizard-28) [inline packages/soundux/soundux.pacscript]:
```suggestion
pacdeps=("pipewire")
```
#13 has been merged.

**Comment by wizard-28** [comment]
@Henryws okay, then so its a problem with the CI.

**Comment by Henryws** [comment]
> @Henryws okay, then so its a problem with the CI.

Yeah, I wouldn't have made a feature if it didn't work

**Review by wizard-28 — APPROVED** [review]
Good to go!

**Pull request merged**